connection.py

```diff
@@ -57,3 +63,10 @@ def send_results(self, results):
             self.sock_send.sendto(
                 f"{results[0]},{results[1]}".encode("utf-8"), (PI_IP, UDP_SEND_PORT)
             )
+
+    def pong(self) -> None:
+        self.ping_time = self.ping.getNumber(0)
+        if abs(self.ping_time - self.old_ping_time) > 0.00000001:
```

Comment:
Replace this magic number with a constant named descriptively, such as TIME_TO_PONG, or whatever makes the most sense.